rooms/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
 
-from .views import RoomsList
+from .views import RoomsList, RoomDetails
 from .apps import RoomsConfig
 
 app_name = RoomsConfig.name
 urlpatterns = [
     path('', RoomsList.as_view(), name='rooms_list'),
+    path('<int:pk>/details', RoomDetails.as_view(), name='room_details')
```

Comment:
It doesn't matter so much with later Django versions, but you should switch the order of these urls. The general advice is that the simplest pattern should be at the bottom, most complicated at the top (I can explain this in more detail sometime)